R/tools.R

```diff
@@ -73,10 +73,16 @@ btw_tools <- function(...) {
     }
   )
 
-  tools_to_keep <- map_lgl(.btw_tools, is_tool_match, tools)
-  res <- .btw_tools[tools_to_keep]
+  tools_is_match <- map_lgl(.btw_tools, is_tool_match, tools)
```

Comment:
Could we rename this variable and maybe the fn it maps over? This didn't have a clear meaning to me before reading through fn's definition

Reply:
Which variable would you suggest renaming?

Reply:
Resolving to make a explicit suggestion